tests/stubs/python-3.12/pybind11-master/numpy-array-use-type-var/demo/_bindings/aliases/foreign_method_arg.pyi

```diff
@@ -5,4 +5,4 @@ import demo._bindings.classes
 __all__ = ["Bar2"]
 
 class Bar2:
-    def set_foo(self) -> int: ...
+    def set_foo(self: demo._bindings.classes.Foo) -> int: ...
```

Comment:
I believe it's a bug in `demo` module. Let's fix it and have a separate set of tests for this PR.